src/base/bittorrent/tracker.cpp

```diff
@@ -203,12 +203,13 @@ Tracker::Tracker(QObject *parent)
 
 bool Tracker::start()
 {
-    const QHostAddress ip = QHostAddress::Any;
     const int port = Preferences::instance()->getTrackerPort();
 
     if (m_server->isListening())
     {
-        if (m_server->serverPort() == port)
+        const int oldPort = m_server->serverPort();
```

Comment:
Maybe not to pollute the outer scope with such declarations?
```c++
        if (const int oldPort = m_server->serverPort()
            ; oldPort == port)
        {
```

Reply:
Done.